Doc/reference/import.rst

```diff
@@ -922,6 +922,53 @@ it is sufficient to raise :exc:`ModuleNotFoundError` directly from
 while raising an exception terminates it immediately.
 
 
+Package Relative Imports
+========================
+
+Relative imports use leading dots. A single leading dot indicates a relative
+import, starting with the current package. Two or more leading dots indicate a
+relative import to the parent(s) of the current package, one level per dot
+after the first. For example, given the following package layout::
+
+    package/
+        __init__.py
+        subpackage1/
+            __init__.py
+            moduleX.py
+            moduleY.py
+        subpackage2/
+            __init__.py
+            moduleZ.py
+        moduleA.py
+
+In either ``subpackage1/moduleX.py`` or ``subpackage1/__init__.py``,
+the following are valid relative imports:
+
+    from .moduleY import spam
+    from .moduleY import spam as ham
+    from . import moduleY
+    from ..subpackage1 import moduleY
+    from ..subpackage2.moduleZ import eggs
+    from ..moduleA import foo
+    from ...package import bar
```

Comment:
You were right to remove the `from ...sys` example, but the `from ...package` example actually won't work either. You can't use this syntax to leave the top-level package, even if you immediately re-enter it. Example:

```
>>> __package__ = "xml.dom"
>>> from . import minidom
>>> from ..etree import ElementTree
>>> from ...xml import etree
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: attempted relative import beyond top-level package
```

Reply:
Thanks for this catch. I have removed that too.